src/modules/auth/credentials/CredentialsCreateContainer.js

```diff
@@ -260,15 +261,29 @@ function CredentialsCreateContainer(props) {
 								</TabPane>
 							</TabContent>
 							<CardFooter>
-								<ButtonWithAuthz
-									color="primary"
-									type="submit"
-									disabled={(activeTab == "create") && (provider == undefined)}
-									resource={resourceCreateCredentials}
-									resources={resources}
-								>
-									{activeTab == "create" ? t('CredentialsCreateContainer|Create credentials') : t('CredentialsCreateContainer|Invite')}
-								</ButtonWithAuthz>
+								{activeTab == "create" ?
```

Comment:
@Samsunggora Not correct, since invitation can also have defined phone as a required field (this info should come from the backend). You have also deliberately removed original button disable `disabled={(activeTab == "create") && (provider == undefined)}` which should stay.

This is how it should be, where `<your watch management>` is basically this `!(watch("email") && watch("phone") && watch("username") && !errors?.phone?.message && !errors?.email?.message)`

```
<ButtonWithAuthz
	color="primary"
	type="submit"
	disabled={(activeTab == "create") && (provider == undefined) || <your watch management>}
	resource={resourceCreateCredentials}
	resources={resources}
>
	{activeTab == "create" ? t('CredentialsCreateContainer|Create credentials') : t('CredentialsCreateContainer|Invite')}
</ButtonWithAuthz>
```